agent/bootstrap/config.go

```diff
@@ -28,6 +28,10 @@ type NewAgentConfig struct {
 
 	// The configured memory backend that stores messages across agent runs.
 	Memory core.MemoryBackend
+
+	// Maximum number of messages restored from memory
+	// default 10
+	MaxMemoryWindowContext int
```

Comment:
Very nice: good addition.

Reply:
This modifications injects the system prompt to context and control de size window of context messages